src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -114,10 +119,33 @@ const TravelAidSection = ({
     fromQuayId,
   );
 
-  useTravelAidAnnouncements({status, focusedEstimatedCall});
+  const selectedEstimatedCall = serviceJourney.estimatedCalls.find(
+    (e) => e.quay.id === fromQuayId,
+  );
+  const situationsForSelected =
+    selectedEstimatedCall?.situations.sort((n1, n2) =>
+      n1.id.localeCompare(n2.id),
+    ) ?? [];
+  const noticesForSelected = selectedEstimatedCall?.notices ?? [];
```

Comment:
For `situations` it is enough (and advised) too only look at `estimatedCall.situations` since this is a collection of all messages on relevant entities (stop place, quay, service joyney, line).

However, this is not the case for `notices`, and other places in the app where we show `notices` we "manually" retrieve them from the different relevant entities, like we do here:
https://github.com/AtB-AS/mittatb-app/blob/cf7423f955ea07cc8b2ac542ddd128ae3b17fc87/src/travel-details-screens/utils.ts#L71

Here is an example of why we needed to do this in our app, where a critical message regarding ferries for Fram where not shown:
https://github.com/AtB-AS/kundevendt/issues/2881

I also posted some relevant info here:
https://mittatb.slack.com/archives/C02EEG7D8EL/p1729842515032949

@jorelosorio @rosvik It might be that you have already decided to not do this extra effort on `notices` in the travel aid screen, but I'm just mentioning it in case there haven't been done a conscious decision on it. And whatever is decided should be reflected in the acceptance criteria or things to test.

Reply:
@gorandalum We did not decide to do it because it took a bit of effort, we intended to be the most precise as possible. So we can resume that for notices we need to use the `getNoticesForServiceJourney` function. Or do we need to do some extra filter?

Reply:
I think the same code can be used, the question is more whether you have the data readily available or not.